src/main.ts

```diff
@@ -508,17 +512,25 @@ export default class WhatsApp {
 
     this.keyPair = null;
 
-    this.qrPath = resolvePath(".", qrPath);
+    this.parameters = {...{
+      keysPath : "./keys.json",
+      qrPath : "./qrcode.png",
+      clientName : "WhatsApp forwarder",
+      clientShortName : "WhatsAppForwarder",
+      restoreSession : false
+    }, ...parameters}
+
+    this.parameters.qrPath = resolvePath(".", this.parameters.qrPath);
 
-    if (restoreSession) {
-      this.keysPath = resolvePath(".", keysPath);
+    if (this.parameters.restoreSession) {
+      this.parameters.keysPath = resolvePath(".", this.parameters.keysPath);
     }
 
-    this.apiSocket.onopen = this.init(loginMsgId, restoreSession);
+    this.apiSocket.onopen = this.init(loginMsgId, this.parameters.restoreSession);
     this.loginMsgId = loginMsgId;
 
-    if (restoreSession) {
-      doesFileExist(this.keysPath!).then(doesExist => {
+    if (this.parameters.restoreSession) {
+      doesFileExist(this.parameters.keysPath!).then(doesExist => {
```

Comment:
```suggestion
    if (this.opts.restoreSession) {
      doesFileExist(this.opts.keysPath!).then(doesExist => {
```
Update property again